content/projects/gsoc/2023/projects/gitlab-plugin-modernization.adoc

```diff
@@ -7,38 +7,47 @@ year: 2023
 status: "Selected"
 sig: platform
 tags:
-- gsoc2022
+- gsoc2023
 - plugins
-//FIXME: create bio for Harsh Pratap Singh in content/_data/authors/
-student: "null_gsoc_contributor"
+student: harsh-ps-2003
 mentors:
 - "markewaite"
 - "krisstern"
 - "freyam"
 links:
-    idea: /projects/gsoc/2023/project-ideas/gitlab-plugin-modernization
-//   gitter: "some gitter channel"
-//   draft: link to the draft/final proposal
-//   meeting: link to internal section of this page with the meeting details and notes
+  idea: /projects/gsoc/2023/project-ideas/gitlab-plugin-modernization
+  gitter: jenkinsci_gitlab-plugin:gitter.im
+  draft: https://docs.google.com/document/d/1fRTWTV2zwLnhWmylavKgQ30apHamrHRfJkK5s_acw7c/edit?usp=sharing
+  meetings: "/projects/gsoc/2023/projects/gitlab-plugin-modernization/#office-hours"
 ---
 
-//above links must be updated
-
 === Abstract
 
-(to be described)
+The GitLab plugin enables Jenkins to seamlessly interact with GitLab allowing it to automatically trigger builds, deploy code, etc. on the basis of GitLab events like Push, Merge Requests and more.
+This project will replace usages of the very old version of the link:https://resteasy.dev/[RESTEasy] library
+with usages of https://github.com/gitlab4j/gitlab4j-api[`gitlab4j-api`]
```

Comment:
One sentence per line as per official recommendation in the CONTRIBUTING.md doc. Same for L29, 30, 31, 32, ad 33. Please revise.